### PR TITLE
Adding secure_accelerator_access AI conformance test instructions

### PR DESCRIPTION
**Pull request by sabbour**

### CODE REVIEW

File: ai-conformance/v1.34/secure_accelerator_access/README.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+Waiting...
+```
+
+### Test 2: Ensure that access to accelerators from within containers is properly isolated.
+
+#### Step 1. Create two Pods, each is allocated an accelerator resource. Execute a command in one Pod to attempt to access the other Pod’s accelerator, and should be denied
+
+This can be verified by running this test [https://github.com/kubernetes/kubernetes/blob/v1.34.1/test/e2e/dra/dra.go#L180](https://github.com/kubernetes/kubernetes/blob/v1.34.1/test/e2e/dra/dra.go#L180) 
+
+With a 1.34 GKE cluster:
```
Comment (ritazh) [inline ai-conformance/v1.34/secure_accelerator_access/README.md]:
```suggestion
With an 1.34 AKS cluster:
```

File: ai-conformance/v1.34/secure_accelerator_access/README.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+ --name gpupool --gpu-driver None -c 1 -s standard_nc96ads_a100_v4
+```
+
+#### Step 3. Install the NVIDIA GPU operator and install the NVIDIA DRA driver
+
+```bash
+az aks get-credentials --resource-group <resource-group> --name <cluster-name>
+
+helm install gpu-operator gpu-operator --repo https://helm.ngc.nvidia.com/nvidia \
+ --version v25.3.3 -n gpu-operator --create-namespace --set devicePlugin.enabled=false
```
Comment (nojnhuh) [inline ai-conformance/v1.34/secure_accelerator_access/README.md]: This will prevent containers from getting access to GPUs by default:

```suggestion
 --version v25.3.3 -n gpu-operator --create-namespace --set devicePlugin.enabled=false --set-string 'toolkit.env[0].name=ACCEPT_NVIDIA_VISIBLE_DEVICES_ENVVAR_WHEN_UNPRIVILEGED,toolkit.env[0].value=false'
```

File: ai-conformance/v1.34/secure_accelerator_access/README.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        exactly:
+          deviceClassName: gpu.nvidia.com
+          allocationMode: ExactCount
+          count: 1
+```
+
+Update the deployment to request a resource:
+
+```yaml
+apiVersion: apps/v1
```
Comment (nojnhuh) [inline ai-conformance/v1.34/secure_accelerator_access/README.md:80]: Here's the pytorch-based manifest I was testing with if we'd rather include that:

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: pytorch-cuda-check
spec:
  replicas: 1
  selector:
    matchLabels:
      app: pytorch-cuda-check
  template:
    metadata:
      labels:
        app: pytorch-cuda-check
    spec:
      containers:
        - name: pytorch-cuda-check
          image: nvcr.io/nvidia/pytorch:25.09-py3
          command: ["/bin/sh", "-c"]
          args:
            - |
              while true; do
                python3 -c "import torch; print(torch.cuda.device_count())"
                sleep 30
              done
          resources:
            claims:
              - name: single-gpu
      resourceClaims:
        - name: single-gpu
          resourceClaimTemplateName: gpu-claim-template
```

File: ai-conformance/v1.34/secure_accelerator_access/README.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+```bash
+kubectl get pods -w
+```
+
+Output:
+
+```output
+NAME                               READY   STATUS              RESTARTS   AGE
+dra-gpu-example-68f595d7dc-mgwqw   1/1     Running             0          3h4m
```
Comment (nojnhuh) [inline ai-conformance/v1.34/secure_accelerator_access/README.md]: This is what the pytorch-based Deployment output looks like:
```
% kubectl get pods -w                                                  
NAME                                  READY   STATUS    RESTARTS   AGE
pytorch-cuda-check-64f4757498-stx82   1/1     Running   0          4m12s
```

File: ai-conformance/v1.34/secure_accelerator_access/README.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+```
+
+
+#### Step 6. Update the Deployment to remove the resource claim request in the Pod spec, the command in the container should fail.
+
+```yaml
+apiVersion: apps/v1
```
Comment (nojnhuh) [inline ai-conformance/v1.34/secure_accelerator_access/README.md:129]: Again for pytorch:

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: pytorch-cuda-check
spec:
  replicas: 1
  selector:
    matchLabels:
      app: pytorch-cuda-check
  template:
    metadata:
      labels:
        app: pytorch-cuda-check
    spec:
      containers:
        - name: pytorch-cuda-check
          image: nvcr.io/nvidia/pytorch:25.09-py3
          command: ["/bin/sh", "-c"]
          args:
            - |
              while true; do
                python3 -c "import torch; print(torch.cuda.device_count())"
                sleep 30
              done
          # resources:
          #   claims:
          #     - name: single-gpu
      resourceClaims:
        - name: single-gpu
          resourceClaimTemplateName: gpu-claim-template
```

File: ai-conformance/v1.34/secure_accelerator_access/README.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+      tolerations:
+      - key: "nvidia.com/gpu"
+        operator: "Exists"
+        effect: "NoSchedule"
+```
+
+Retrieve the logs, they should show a failure:
+
+```bash
+kubectl logs dra-gpu-example-5c97694b59-5tmqr
```
Comment (nojnhuh) [inline ai-conformance/v1.34/secure_accelerator_access/README.md]: pytorch:
```
kubectl logs pytorch-cuda-check-64f4757498-n82f6
```

File: ai-conformance/v1.34/secure_accelerator_access/README.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+```
+
+Output:
+
+```output
+Fri Oct 10 01:12:14 UTC 2025
+bash: line 1: nvidia-smi: command not found
+Waiting...
```
Comment (nojnhuh) [inline ai-conformance/v1.34/secure_accelerator_access/README.md]: pytorch:
```
% kubectl logs pytorch-cuda-check-64f4757498-n82f6
/usr/local/lib/python3.12/dist-packages/torch/cuda/__init__.py:63: FutureWarning: The pynvml package is deprecated. Please install nvidia-ml-py instead. If you did not install pynvml directly, please report this to the maintainers of the package that installed pynvml for you.
  import pynvml  # type: ignore[import]
0
```

File: ai-conformance/v1.34/secure_accelerator_access/README.md [hunk 8]
```diff
@@ -176,7 +178,7 @@ Waiting...
 
 This can be verified by running this test [https://github.com/kubernetes/kubernetes/blob/v1.34.1/test/e2e/dra/dra.go#L180](https://github.com/kubernetes/kubernetes/blob/v1.34.1/test/e2e/dra/dra.go#L180) 
 
-With a 1.34 GKE cluster:
+With an 1.34 AKS cluster:
```
Comment (nojnhuh) [inline ai-conformance/v1.34/secure_accelerator_access/README.md]:
```suggestion
With a 1.34 AKS cluster:
```
